cluster/addons/dashboard/dashboard-rbac.yaml

```diff
@@ -11,6 +11,10 @@ rules:
 - apiGroups: [""]
   resources: ["secrets"]
   verbs: ["create"]
+  # Allow Dashboard to create 'kubernetes-dashboard-settings' config map.
```

Comment:
is there a reason not to let this be created by the addon manager... blanket create of configmaps in kube-system is really powerful. the same request was made for secrets when updating to 1.8.0 (https://github.com/kubernetes/kubernetes/pull/53046#discussion_r153808146), and the indication was that the dashboard would update to do that.

Reply:
https://github.com/kubernetes/dashboard/pull/2667